Spring-Project/src/main/java/com/epam/project/spring/controller/ProductController.java

```diff
@@ -0,0 +1,47 @@
+package com.epam.project.spring.controller;
+
+import com.epam.project.spring.dto.ProductDto;
+import com.epam.project.spring.service.ProductService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
```

Comment:
Hi Zenyk. To avoid duplication please remove "/product" from path of the all **@*Mapping** on the method level and add `@RequestMapping("/product")` on the Controller level

Reply:
Thanks. Resolved